channels/ON-CHAIN.md

```diff
@@ -227,11 +274,13 @@ The `channel_settle` MUST only be included in a block if:
 Serialization defined [here](../serializations.md#channel-settle-transaction)
 
 
-- `channel_id`:
-- `from`
+- `channel_id`: channel id as recorded on-chain
+- `from`: participant of the channel that posts the slashing transaction
```

Comment:
Is "slashing" here a copy-paste error?

Based on https://github.com/aeternity/protocol/blame/c43ae3cacc991e1a12b42210ea8ed27e496b442c/channels/README.md#L333-L342 the channel_settle can be preceded by a channel_close_solo without any channel_slash.

Reply:
copy-paste error, indeed